prow/cmd/crier/main.go

```diff
@@ -196,13 +200,22 @@ func main() {
 	signal.Notify(sigTerm, syscall.SIGTERM)
 	signal.Notify(sigTerm, syscall.SIGINT)
 
-	go func() {
-		<-sigTerm
-		logrus.Info("Crier received a ternimation signal and is shutting down...")
+	<-sigTerm
```

Comment:
we might not always get sigTerm

Reply:
when terminating the pod?

Reply:
we probably should, but linux doesn't actually guarantee this. I can still just send SIGKILL afaik. SIGTERM is ~ for nicely telling a process it should please quit now.

Reply:
Processes do not get to handle SIGKILL.  The kernel never delivers that signal to a process, it just kills it. I think handling SIGTERM + SIGINT is sufficient.